PC/pyconfig.h

```diff
@@ -739,4 +739,7 @@ Py_NO_ENABLE_SHARED to find out.  Also support MS_NO_COREDLL for b/w compat */
 /* Define if libssl has X509_VERIFY_PARAM_set1_host and related function */
 #define HAVE_X509_VERIFY_PARAM_SET1_HOST 1
 
+/* Define if you want to disable the GIL */
+#undef Py_GIL_DISABLED
```

Comment:
Do you want to rename the `PC/pyconfig.h`? It seems like some tools might treat the input template as if it's the true `pyconfig.h` file.

Reply:
It's not actually a template, it's a totally valid and usable header file. In the default case (for now), `Py_GIL_DISABLED` is undefined, and we just change that on build if requested.

For editing in an IDE, we need some kind of actual header file available or else all the code assistance tools will fail. Rather than requiring the IDE understand the build directory (which VS can do, but others may not), this allows us to have a default `GIL_DISABLED` header before building and then the correct one when actually building.